RELEASE_NOTES.md

```diff
@@ -27,240 +27,185 @@ will not be possible to downgrade from 3.0 to 2.6.x any more.
 
 ## The Big Ones
 
-- A full rework of the GUI Gtk/CSS. The whole GUI is now fully
-  controlled by CSS rules. There is no more size, color, position in
+- A full rework of the GUI. The whole GUI is now fully
+  controlled by [GTK+ CSS rules](https://developer.gnome.org/gtk3/stable/chap-css-overview.html). There is no more size, color, position in
   Gtk C code. This makes the whole GUI themable. This version comes
-  with different theme:
+  with several themes:
 
      - darktable                : the default theme
 
      - darktable-elegant-darker : more condensed fonts
-                                  best experience if Roboto font installed
+                                  best experience with Roboto font installed
 
      - darktable-icons-darker   : as elegant, with IOP icons
-                                  best experience if Roboto font installed
+                                  best experience with Roboto font installed
 
      - darktable-elegant-dark   : lighter version
      - darktable-elegant-grey   : even lighter version
 
-     - darktable-icons-dark     : lighter version
-     - darktable-icons-grey     : even lighter version
+     - darktable-icons-dark     : lighter version. with IOP icons
+     - darktable-icons-grey     : even lighter version, with IOP icons
 
-  Note that this requires Gtk 3.22 or higher to work properly.
+  Note that the new GUI requires Gtk+ 3.22 or higher to work properly.
 
 - The color picker on the blend, tone curve, color zones and fill
-  light modules now allows to select an area by <kbd>Ctrl+click</kbd> on it.
+  light modules now allows you to select an area when enabled by
+  using <kbd>Ctrl+click</kbd> on the button.
 
```

Comment:
What is color picker on blend (module)?
The fill light module picker on exposure need the ctrl+click help text to let users know you can select area by area

Reply:
> What is color picker on blend (module)?

Good catch, I corrected that in another part of the document, but somehow let it slide there... changed to be "parametric mask controls" in the other case.